website/content/en/docs/golang/legacy/quickstart.md

```diff
@@ -1,9 +1,11 @@
 ---
-title: Golang Based Operator Quickstart
+title: Golang Based Operator Quickstart (Legacy)
 linkTitle: Quickstart
 weight: 2
 ---
 
+**Note:** This guide is for the legacy CLI and project layout. See the [new docs][new_docs] for the [Kubebuilder aligned CLI][new_CLI] and project layout.
```

Comment:
Given that this quickstart assumes `operator-sdk new --type=go` exists, should we remove it?

This is non-blocking and could be done in a follow-up.

Reply:
Oh yeah forgot about that.
There's some stuff in the FAQ talking about metrics as well.
I'll add a TODO to follow this up.